src/main/java/seedu/address/logic/commands/DeleteCommand.java

```diff
@@ -42,7 +42,7 @@ public CommandResult execute(Model model) throws CommandException {
 
         Person personToDelete = lastShownList.get(targetIndex.getZeroBased());
         model.deletePerson(personToDelete);
-        return new CommandResult(String.format(MESSAGE_DELETE_PERSON_SUCCESS, Messages.format(personToDelete)));
```

Comment:
I think it's better to print the details of the person instead of just the name.